op-e2e/e2eutils/disputegame/split_game_helper.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const defaultTimeout = 5 * time.Minute
+const defaultTimeout = 20 * time.Minute
```

Comment:
ouch - is it really that much slower?

Reply:
Yeah. Because the max depth is much longer. I'm considering adding a fast auto-resolver for local development. Because it takes over 10 minutes for the challenger to resolve a game.

Reply:
oh, we didn't need to increase the max depth in production - is it just that we need to reduce it for super games in e2e tests?  We originally had to increase it to support the 1024 steps but have since reduced that.

Reply:
The Super DG tests use the same max depth as production. But it turns out that the FDG tests use a much smaller max depth than production. So it's relatively slower to run the new tests.
In retrospect, it's better this way; having tests that match production configs.

Reply:
I just tried running the `TestSuperCannonRootBeyondProposedBlock_ValidRoot` to see why it was failing and it's taking 30 minutes before it fails.  I feel like that's just way too long.  It was that test length that caused us to reduce the max depth for op-e2e originally and I think it's a worthwhile trade off here as well.  The extra depth doesn't really change anything as long as it's still odd or even at the split and max depths. And even that doesn't hugely matter I don't think given that we test both valid and invalid roots so the honest and dishonest actors are tested for both odd and even levels.

Reply:
We don't necessarily need to block this PR on reducing the game depth, but I do think it's worth doing to make the testing cycle faster.

Reply:
You're probably right. Will see if I feel differently after working on the last bit of tests with a shorter depth.